spectree/models.py

```diff
@@ -1,7 +1,11 @@
+import re
 from enum import Enum
 from typing import Any, Dict, Sequence
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, validator
+
+# OpenAPI names validation regexp
+OAPI_NAME_RE = re.compile(r"^[A-Za-z0-9-._]+")
```

Comment:
Can choose one from (`OpenAPI_NAME_RE`, `OPENAPI_NAME_RE`). `OAPI_NAME_RE` is a little bit confusing.